agent/sidecar_service_test.go

```diff
@@ -63,18 +63,8 @@ func TestAgent_sidecarServiceFromNodeService(t *testing.T) {
 					LocalServicePort:       1111,
 				},
 			},
-			wantChecks: []*structs.CheckType{
-				{
-					Name:     "Connect Sidecar Listening",
-					TCP:      "127.0.0.1:2222",
-					Interval: 10 * time.Second,
-				},
-				{
-					Name:         "Connect Sidecar Aliasing web1",
-					AliasService: "web1",
-				},
-			},
```

Comment:
Since these checks are removed, shall we add somewhere to verify the post assignment results? I'm think somewhere testing `func (a *Agent) addServiceLocked(req addServiceLockedRequest)`, we need to verify the port is assigned as expected.

Reply:
done. added verification in TestAgent_loadServices_sidecar.